### DISCUSSION TITLE
Contributing LSP4E to SimRel

### DISCUSSION THREAD

**Discussion by jonahgraham** [eclipse/lsp4e#790] [General]
The recent release of LSP4E was not contributed to SimRel. Was this done on purpose or simply an oversight? If just an oversight I can do the simrel contribution.

Is LSP4E going to continue to be contributed to SimRel directly going forward, or is there a plan to change it to a "library" type project that is contributed by the dependent projects. I see that [WWD](https://download.eclipse.org/wildwebdeveloper/releases/1.3.0/) and [LinuxTools](https://download.eclipse.org/linuxtools/updates-docker-nightly/plugins/) contain the LSP4E plugins in their repo, so perhaps CDT LSP should be doing the same?

For example the CDT LSP project would like to release a preliminary version to be installed in 2023-09 so it would be helpful. The version of LSP4E we need is the recently released 0.24.1 version (https://github.com/eclipse-cdt/cdt-lsp/pull/195)

Thoughts or comments?

**Comment by mickaelistria** [comment]
It is an oversight. I don't know if someone actually takes close care of the SimRel contribution nowadays (I personally now only work with snapshots of everything, never use SimRel anymore so always forget about maintaining it).
If someone wishes it to stay in SimRel, that's fine if they maintain the entry. If no-one cares about maintaining it, we can just remove it from SimRel and recommend consumers to include it in their p2 repo as a transitive deps. SimRel will include it as a transitive dependency as well (without it being categorized and so on through).

**Reply by jonahgraham** [reply]
OK - I'll handle SimRel this time.

**Reply by rubenporras** [reply]
Mainly out of curiosity since you will already take care of it (I wonder because I did the release), what needs to be done to contribute to SimRel?

**Reply by akurtakov** [reply]
Opening a Gerrit like this one https://git.eclipse.org/r/c/simrel/org.eclipse.simrel.build/+/204117
